src/chrome/content/rules/Maybank2u.com.ph.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+	For other Maybank coverage, see Maybank.com.xml
+-->
+<ruleset name="Maybank2u.com.ph">
+
+	<target host="maybank2u.com.ph" />
+	<target host="www.maybank2u.com.ph" />
+	
+	<securecookie host=".+" name=".+" />
+
+	<!-- only main website; RIBNOW, resources and ph_maybank_gif directories are functional -->
+	<rule from="^http://(www\.)?maybank2u\.com\.ph/$"
```

Comment:
I get a 404 with this rule enabled.

Reply:
WFM and travis

Reply:
@Bisaloo or @gloomy-ghost can you confirm this?

Reply:
I get 404 as well.